Source/JavaScriptCore/bytecode/UnlinkedCodeBlock.cpp

```diff
@@ -105,7 +105,8 @@ void UnlinkedCodeBlock::visitChildrenImpl(JSCell* cell, Visitor& visitor)
         visitor.append(barrier);
     for (auto& barrier : thisObject->m_functionExprs)
         visitor.append(barrier);
-    visitor.appendValues(thisObject->m_constantRegisters.data(), thisObject->m_constantRegisters.size());
+    auto constantRegisters = thisObject->m_constantRegisters.span();
+    visitor.appendValues(constantRegisters.data(), constantRegisters.size());
```

Comment:
While this change is clearly an improvement, I do think that giving the slot visitors an overload that takes a container would be even nicer and avoid the intermediate span. 

There already exists one that takes begin/end iterators, so this could also be changed to:

```
 visitor.append(thisObject->m_constantRegisters.begin(), thisObject->m_constantRegisters.end());
```

(not something you need to change for this PR, just pointing it out).